src/sqs/backfill-discovery.test.ts

```diff
@@ -67,7 +67,7 @@ describe("Discovery Queue Test - GitHub Client", () => {
 			gitHubAppId: gitHubServerApp.id
 		});
 
-		await sqsQueues.backfill.start();
+		sqsQueues.backfill.start();
```

Comment:
TBH this change looks quite bizarre. In one place we are adding await, in another we are removing it, exactly for the same method...

Reply:
This is clean up.  This function had previously been changed to not be async.  When that change was made these calls were not updated to remove the awaits here that no longer do anything.  They are awaiting a void instead the previous Promise<void>